qiskit/_util.py

```diff
@@ -51,14 +50,16 @@ def _check_ibmqe_version():
     # Find the IBMQuantumExperience version specified in qiskit.
     if qiskit_local:
         try:
-            # Mock setuptools.setup, as it would be executed during the import.
-            with patch('setuptools.setup'):
-                # Use a local import to fall back gracefully if not present.
-                from setup import requirements
-                ibmqe_require_line = next(r for r in requirements if
-                                          r.startswith('IBMQuantumExperience'))
+            with open('requirements.txt') as reqfile:
+                for line in reqfile:
+                    if 'IBMQuantumExperience' in line:
+                        ibmqe_require_line = 'IBMQuantumExperience>=1.8.26\n'
+                        break
+                else:
+                    raise pkg_resources.RequirementParseError
```

Comment:
Can we try to make this loop/check more pythonic, and also make sure that the requirement line is read from the file (as currently you are hard-coding the 1.8.26 version in the code, which kind of defeats the purpose? Perhaps something along the lines of:

```python
    with open('requirements.txt') as reqfile:
        ibmq_require_line = next(line for line in reqfile if line.startswith('IBMQuantumExperience'))
        ibmqe_require = pkg_resources.Requirement(ibmqe_require_line)
```

The `next()` might raise a `StopIteration` that can be caught by the next `except` directly. It might be needed to chop the newline or revise the path to `requirements.txt` - can you double check it is being read and parsed correctly after the changes?

Reply:
your suggestion worked as expected. The hardcoded line was a mistake. Fixed now.